### DISCUSSION TITLE
In order to make a better translation.

### DISCUSSION THREAD

**Discussion by laramaceo** [tnodir/fort#310] [General]
Sorry, I've had doubts about how to translate this sentence since the beginning of this work.
“Tray Menu Flags”.
I have translated the word “Flags” as “icon”. 
Icon in the system tray.
I am not sure but I think Spanish speakers do not use the word “Flag”.

I don't understand the relationship between: Action Confirmations and Tray Menu Flags.
I do understand this case: : Action Confirmations and Quit.
When I try to close the program from the system tray, a confirmation message appears asking me if I want to quit Fort .
Can you clarify the question for me?
 Can you give me a visual reference of the relationship between:  Action Confirmations and Tray Menu Flags?
I want to be sure if: “Icon in the system tray” is the best Spanish translation. 

I feel quite satisfied with my work done, but if someone with knowledge of Spanish language, networking and a little knowledge of graphic design; comes and wants to improve my work; I will be happy.

**Comment by tnodir** [comment]
“Tray Menu Flags” means "Check-able options in the tray icon's context menu".

**Comment by tnodir** [comment]
> I don't understand the relationship between: Action Confirmations and Tray Menu Flags.

If you check the "Tray Menu Flags", then Fort will ask confirmation for any operations from the tray icon's context menu.

E.g. if you click on "Filter enabled" flag, then a "Are you sure..." dialog will be opened.

**Comment by tnodir** [comment]
Maybe we should rename the "Tray Menu Flags" to "Operations from Tray Menu"?

Because it affects a "Block Traffic" and "Filter Mode" lists too.

**Reply by tnodir** [reply]
Done.
